### DISCUSSION TITLE
Video does not support audio?

### DISCUSSION THREAD

**Discussion by MuhammadEhsan2** [Serene-Arc/bulk-downloader-for-reddit#310] [Q&A] [answered]
When I download videos, it does not support audio, please tell me how to solve this issue.

**Comment by aliparlakci** [comment, marked as answer]
Open a proper issue, not a discussion. Please read this first: [Opening an issue](https://github.com/aliparlakci/bulk-downloader-for-reddit/blob/master/docs/CONTRIBUTING.md#opening-an-issue)
